test/bin/vault/test_main.py

```diff
@@ -95,6 +96,54 @@ def test_keep_files(self, mock_add, mock_remove):
         mock_add.assert_called_with(Branch.Keep, [T.Path("/file1"), T.Path("/file2")])
         mock_remove.assert_not_called()
 
+    # Test for log warning message about symlink
+    @mock.patch('bin.vault.untrack')
+    @mock.patch('bin.vault.add')
+    def test_keep_files_symlink(self, mock_add, mock_remove):
+        
+        self._tmp = TemporaryDirectory()
+        path = T.Path(self._tmp.name).resolve()
+        # Form a directory hierarchy
+        filepath = path / "a"
+        symlink= path / "b"
+        filepath.touch()
+        os.symlink(filepath, symlink)
+      
+        main(["__init__","keep" , str(symlink)])
+        mock_add.assert_called_with(Branch.Keep, [filepath])
+        mock_remove.assert_not_called()
+        self._tmp.cleanup()
+
+    # Test for log warning message about symlink
```

Comment:
Is this supposed to be the same comment as line 99?

Reply:
More or less, yeah, but good point, better to be more specific.